### ISSUE TITLE
Unreal Engine 5.5.1 Editor crashes instantly when using Vulkan RHI

### ISSUE THREAD

**Issue by Zum0DePapaya** [IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT#919]
### Checklist [README]

- [X] Device is not undervolted nor overclocked
- [X] Device is using the [latest drivers](https://github.com/IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT/wiki/Intel-driver-updates-and-changelogs)
- [X] Application is not cracked, modded and use the latest patch

### Application [Required]

Unreal Engine 5.5.1

### Processor / Processor Number [Required]

AMD Ryzen 7 3700X

### Graphic Card [Required]

Intel Arc B580 Challenger 12GB OC

### GPU Driver Version [Required]

32.0.101.6332_101.6253

### Rendering API [Required]

- [X] Vulkan
- [ ] OpenGL
- [ ] DirectX12
- [ ] DirectX11
- [ ] DirectX10
- [ ] DirectX9
- [ ] Not applicable

### Windows Build [Required]

Windows 10 22H2

### Intel System Support Utility report

[igcit_ssu.txt](https://github.com/user-attachments/files/18266868/igcit_ssu.txt)

### Description and steps to reproduce [Required]

Simply opening any Unreal Engine 5.5.1 project using the Vulkan RHI crashes instantly upon loading, disabling things like raytracing or lumen have no effect and the result is the same.

1. Using the latest Unreal Engine version create a new Unreal Engine Project using any template or none at all.

2. When it finishes loading open the project settings and change the Default HRI option to Vulkan.

3. You will be asked to restart the editor for the changes to take effect, press the restart now button.

4. The program will crash instantly when the editor finishes loading.

![VulkanCrash](https://github.com/user-attachments/assets/fb727f20-a4bc-4c68-b9f3-d6a8b9121b92)

### Crash dumps [Required, if applicable]

[UECC-Windows-9111705943226FEE870BA0A87A425135_0000.zip](https://github.com/user-attachments/files/18266890/UECC-Windows-9111705943226FEE870BA0A87A425135_0000.zip)

**Comment by Arturo-Intel** [comment]
Hello @Zum0DePapaya 

I try to repro this behavior in our lab without any luck

I follow the next steps:
1. Install UE 5.5.1
2. Created an empty project
3. Once the editor open (took a long time to open) I change from default to Vulkan the RHI in the Project Setting page
![image](https://github.com/user-attachments/assets/d63ca550-62bc-42cb-b383-00d62dec7e76)
4. Close the editor and reopen the project

I took a while to open the project using the RHI targeted to vulkan (like 5 to 8 min) but at the end I was able to open the editor without any crashes.

The only diff I see is that I am using Window 11 23H instead of Windows 10
Also, **I am using a newer version of the driver (32.0.101.6332)**

Can you please try with the latest driver version? (https://www.intel.com/content/www/us/en/download/785597/intel-arc-iris-xe-graphics-windows.html) 

--r2

**Comment by Zum0DePapaya** [comment]
Sorry for the late response, even with the latest drivers I still have the same issue, no idea why. Also unless I'm confused the latest b580 drivers should be 32.0.101.101.6256.

**Comment by SavageWaffle** [comment]
Fixes TBD

**Comment by Felipe-Intel** [comment]
@Zum0DePapaya yes, that is the latest driver for B580 cards. @Arturo-Intel will follow up with you soon. 
@SavageWaffle this is still under investigation.
